### PR TITLE
Fix  #3091: WebSocketMessageTransport.ReceiveAsync null return value isn't handled

### LINKED ISSUE #3091: WebSocketMessageTransport.ReceiveAsync null return value isn't handled
It is possible that new socket become closed before first request to it was made. `RHost.RunWorker` should handle possible null value and exit without throwing `InvalidDataException`.

### PR DESCRIPTION
**Pull request by AlexanderSher**
RHost can fail at any time after start, including the moment when RTVS requests the handshake.

### CODE REVIEW

File: src/Host/Client/Impl/Host/RHost.cs [hunk 1]
```diff
@@ -81,6 +81,8 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator, IRBlobSe
 
             if (message != null) {
                 Log.Response(message.ToString(), _rLoopDepth);
+            } else {
+                Log.Response(_transport.CloseStatusDescription, _rLoopDepth);
```
Comment (MikhailArkhipov) [inline src/Host/Client/Impl/Host/RHost.cs]: Does it handle null?

Comment (MikhailArkhipov) [inline src/Host/Client/Impl/Host/RHost.cs]: Oh, nvm it is at least empty

**Comment by MikhailArkhipov** [comment]
👍

File: src/Host/Client/Impl/Host/RHost.cs (hunk 1 above)
Comment (AlexanderSher) [inline src/Host/Client/Impl/Host/RHost.cs]: Yes, we return `null` when transport is closed

**Pull request merged**